docs/src/user-guide/install.md

```diff
@@ -1,40 +1,111 @@
+| CPU     | x86_64 (Intel, AMD) processor with at least 4 physical cores     |
+| RAM     | 16GB DDR4     |
+| Storage     | at least 60GB SSD (NVMe SSD is recommended. HDD will be enough for localnet only)    |
+
+There are different ways to install Namada: [From Source](#from-source), [From Binaries](#from-binaries), [From Docker](#from-docker) and [From Nix](#from-nix).
+
+### From Source
+
+If you'd like to install Namada from source you will have to install some dependencies first: [Rust](https://www.rust-lang.org/tools/install), [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), Clang, OpenSSL and LLVM.
+
+First install Rust.
+```bash
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+At the end of the installation, make sure that Cargo's bin directory ($HOME/.cargo/bin) is available on your PATH environment variable. You can either restart your shell or run `source $HOME/.cargo/env` to continue.
 
-apt-get install -y make git-core libssl-dev pkg-config libclang-12-dev
+If you already have Rust installed, make sure you're using the latest version by running:
+```bash
+rustup update
 ```
+Then, install the remaining dependencies.
 
-Last, to build and install anoma you can run the following command:
+**Ubuntu:** running the following command should install everything needed:
+```bash
+sudo apt-get install -y make git-core libssl-dev pkg-config libclang-12-dev
+```
+**Mac:** installing the Xcode command line tools should provide you with almost everything you need:
+```bash
+xcode-select --install
+```
+Now, that you have all dependencies installed you can clone the source code from the [Anoma repository](https://github.com/anoma/anoma) and build it with:
 
 ```bash
+git clone https://github.com/anoma/anoma.git --single-branch --branch $ANOMA_TESTNET_BRANCH
+cd anoma 
 make install
 ```
 
-## Nix
+Once done, you can go to [Setting up Namada](../testnets/internal-testnet-1/#setting-up-namada) section.
+
+### From Binaries
+
+>⚠️ During internal and private testnets, links to releases provided in this section might be outdated and inconsistent to the current testnet.
+>To avoid any issues, we recommend you [build from source](#build-from-source).
+
+If you'd like to install Namada from binaries you will have to install some dependencies first: [Rust](https://www.rust-lang.org/tools/install), [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Tendermint](https://docs.tendermint.com/master/introduction/install.html) `0.34.x` and GLIBC `v2.29` or higher.
+
+First install Rust.
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+At the end of the installation, make sure that Cargo's bin directory ($HOME/.cargo/bin) is available on your PATH environment variable. You can either restart your shell or run `source $HOME/.cargo/env` to continue.
+
+If you already have Rust installed, make sure you're using the latest version by running:
+```bash
+rustup update
+```
```

Comment:
Rust shouldn't be needed for running the binaries?

glibc requirement is only for Linux I think, we should section it out. On my setup I've been able to compile and run everything without glibc installed.